### ISSUE TITLE
File close option in Steamshovel app (Trac #1025)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1019]
Hello, it would be nice if the steamshovel app included an option to close a file. Currently (I think) one has to quit the app to close everything.

Migrated from https://code.icecube.wisc.edu/ticket/1025
```json
{
    "status": "closed", 
    "changetime": "2015-07-06T16:43:57", 
    "description": "Hello, it would be nice if the steamshovel app included an option to close a file. Currently (I think) one has to quit the app to close everything.\n", 
    "reporter": "delia.tosi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1436201037705648", 
    "component": "other", 
    "summary": "File close option in Steamshovel app", 
    "priority": "normal", 
    "keywords": "steamshovel", 
    "time": "2015-06-22T19:55:44", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150623T15:42:32: 
* delia.tosi changed owner from "" to "hdembinski"
* delia.tosi changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150623T15:53:22: hdembinski commented:

> What exactly do you need?
> 
> Moderate effort: Closing all currently opened files.
> Moderate effort: Closing the file that was opened last.
> High effort: Closing an arbitrary file in the frame sequence.
> 
> I will resist the last option, unless a very good case is made. There is no good reason not to close and re-open Steamshovel, especially since the session is saved these days.

**Comment by icecube-trac** [comment]
Trac update at 20150706T16:43:57: 
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 134784:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="134784"
> added functionality to close open files, closes #1019
> ```


* hdembinski changed status from "assigned" to "closed"
